torchdata/datapipes/iter/__init__.py

```diff
@@ -31,6 +31,11 @@
     IoPathFileLoaderIterDataPipe as IoPathFileLoader,
     IoPathSaverIterDataPipe as IoPathSaver,
 )
+from torchdata.datapipes.iter.load.fsspec import (
+    FSSpecFileListerIterDataPipe as FSSpecFileLister,
+    FSSpecFileLoaderIterDataPipe as FSSpecFileLoader,
+    FSSpecSaverIterDataPipe as FSSpecSaver,
+)
```

Comment:
Could you please also add these class to `__all__`?

Reply:
done